ui/widgets/autocomplete.js

```diff
@@ -267,8 +268,11 @@ $.widget( "ui.autocomplete", {
 				// Announce the value in the liveRegion
 				label = ui.item.attr( "aria-label" ) || item.value;
 				if ( label && String.prototype.trim.call( label ).length ) {
-					this.liveRegion.children().hide();
-					$( "<div>" ).text( label ).appendTo( this.liveRegion );
+					clearTimeout( this.liveRegionTimer );
+					var that = this;
+					this.liveRegionTimer = setTimeout( function() {
+						that.liveRegion.html( $( "<div>" ).text( label ) );
+					}, 400 );
```

Comment:
`400` is quite a big delay, can we use 100? See https://web.dev/rail/. Anything higher than 100 doesn't feel as immediate to the user.

Reply:
Changed to 100